source/Irrlicht/COpenGLDriver.h

```diff
@@ -217,6 +722,609 @@ class COpenGLDriver final : public CNullDriver, public COpenGLExtensionHandler
 	public:
         struct SAuxContext;
 
+		enum EOpenGLFeatures {
+			IRR_3DFX_multisample = 0,
+			IRR_3DFX_tbuffer,
+			IRR_3DFX_texture_compression_FXT1,

```

Comment:
this is on the border of making our code a little illegible, maybe create a class `src/irr/video/COpenGLFeatureMap` and shove all this extension stuff there and then make `COpenGLDriver` inherit from it?